### ISSUE TITLE
must change password twice before login is possible

### ISSUE THREAD

**Issue by feenes** [mhcomm/django-pwdtk#5]
reporter: external user
setup: python 2.7 django1.8:

pwdtk is behaving a little strange. Let me explain.
1. Password expires.
2. User is redirected to change password page.
3. User renews the password
3. The new delay is set properly in database.
4. When the user logs in again, he's redirected to change password page even if the PwdTkMustChangePassword is not raised.
5. So user updates the password again.
6. Then the user is able to login.

Basically, the user requires to renew the password twice. There's something wrong with the login_func in watch_login function in auth_backends.py

**Comment by feenes** [comment]
Tried to reproduce with the test project.
Cannot reproduce:
1.) start runserver with a project, that as an admin user with a password
2.) go to /
3.) click on login
4.) click on "access the protected space" (Should not fail, login was successful)
5.) go to /admin/password_change/
6.) enter old password + twice new password
7.) go to / (or alternatively click on "Home" and then on "View Site")
8.) click on "Login"
9.) Enter username / password (you should get welcome to the test space <username>)
10.) click on "access the protected space"

**Comment by klausfmh** [comment]
issue solved in django project. in the given project a logout after password change fixed the issue
